demo/index.html

```diff
@@ -25,6 +25,8 @@
       --shareLinkColor: #fff;
       --shareIconBorder: #fff;
       --shareIconBg: #151515;
+      --closeMenuIconWidth: 18px;
```

Comment:
same here. let's keep this component as simple as possible with a default padding to compensate for the button

Reply:
We need to ensure we're keeping the contents of the header elements in these submenus from interfering with the close icon. Since the close icon is not in this component's direct control anymore, we have to read in the close icon's width and set a padding accordingly. Retaining this for all packages.

Reply:
I would like to continue to push back for more simplicity.
can we confer with @pezvi around design to ensure that there is space?

Reply:
The way it's built now, when the icon width changes, there's one point of update across (at the moment) 5 packages. Hard-coding a padding in each of these submenus means there's an update necessary in the menu slider package as well as all of these submenu packages. Why would we go that route?

Reply:
![image](https://user-images.githubusercontent.com/7840857/92507141-5ecea200-f1bb-11ea-9067-a5eafb7a0e2d.png)
This is the space that we are talking about.
Don't hard code, just make a width that doesn't go to the end of submenu, right?
If it doesn't fit, then let's wrap.

Why go the route of having so much styling values to accommodate a thing that the menu isn't responsible for?

Reply:
These components have no other way to know what area the close button is going to take up than to receive them via CSS variables. There's no way to set a width of the header that will ensure it expands to fit its parent and also keeps out of the way of the close button apart from calculating it in CSS, like with `width: calc(100% - 18px)`. You still have a hard-coded value, and you're putting padding in your width calculation when you should be using the padding property instead.

Reply:
width: 80%?

Reply:
In wider widths, that will leave a large unused space between the header and close button. In narrow widths, if the icon changes size, it still may interfere.

Reply:
That should not be a problem. 😃 
(we gotta see the whole thing in concerto anyway, before we fully release.)

The concern I have is that though we yanked out the close button, we continue to have an "understanding" in this component about a close button.
let's simplify and make sure that this component is only responsible for itself.

I would also like to note that this convention that we are setting will permeate through to other submenus that we create. That is why I am fighting to keep their responsibility as simple as possible.

Then we can continue to tackle weaving the main book navigator into item navigator - which is where we need the real muscle 💪

Reply:
hi! let's hold on the work on this & the others in question - I've got something that will make their lives even simpler.  thanks!! @shaneriley